The amr object is missing the 'sc' value which is set when the user logs in with Smart Card (#1654)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

| `kba`    | Knowledge-based authentication                         | Security Question factor                                                                               |
| `mfa`    | Multiple-factor authentication                         | This value is present whenever any MFA factor verification is performed.                               |
| `mca`    | Multiple-channel authentication                        | Authentication requires communication over more than one channel, such as Internet and mobile network. |

> | `sc`    | Smart Card authentication                                | This value is present whenever the user authenticated using smart card, such as personal identity verification (PIV) card or Common Access Card (CAC)                                                                          |

Reviewer: ```suggestion
| `sc`    | Smart card authentication                                | User authenticated using a smart card, such as a Personal Identity Verification (PIV) card or Common Access Card (CAC)                                                                          |
```
